<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTemDlg</class>
 <widget class="QDialog" name="NewTemDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>374</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>55</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_34">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_35">
       <item>
        <widget class="QLabel" name="labelPN_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>成品编号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EditPN">
         <property name="minimumSize">
          <size>
           <width>184</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>260013</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_36">
       <item>
        <widget class="QLabel" name="labelPartCode_4">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Fanout PN：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FanoutPN">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>A031378AA</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_42">
     <item>
      <layout class="QHBoxLayout" name="layoutFiberSpec_7">
       <item>
        <widget class="QLabel" name="labelFiberSpec_7">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>光纤规格：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="FiberSpec">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>184</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>OS2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OM4 Aqua</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OM4 HV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OM5</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutWave_7">
       <item>
        <widget class="QLabel" name="labelWave_7">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>波类型与数值：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="WaveLambda">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>SM 1310/1550</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MM 850/1300</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="saveBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_43">
     <item>
      <layout class="QHBoxLayout" name="layoutFiberSpec_8">
       <item>
        <widget class="QLabel" name="labelFiberSpec_8">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string> QR模板 ：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="QRTem">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>184</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Template A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Template B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Template C</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_20">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutWave_8">
       <item>
        <widget class="QLabel" name="labelWave_8">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>标贴模板：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="LabelTem">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Template 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Template 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Template 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Template 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Template 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Template 6</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_41">
     <item>
      <layout class="QHBoxLayout" name="layoutPolarity_4">
       <item>
        <widget class="QLabel" name="labelPolarity_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>极性类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Polarity">
         <property name="minimumSize">
          <size>
           <width>184</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>A/C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Universal</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_18">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutFiberCount_4">
       <item>
        <widget class="QLabel" name="labelIL_4">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>最大损耗值IL：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ILMax">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="suffix">
          <string> dB</string>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="cancelBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_37">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_38">
       <item>
        <widget class="QLabel" name="labelInterface_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>接口类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_39">
         <item>
          <widget class="QComboBox" name="Interface">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>MTP-LC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MTP-SN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MTP-MDC</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="FiberCount">
           <property name="minimumSize">
            <size>
             <width>77</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>8F</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12F</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16F</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_17">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>45</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_40">
       <item>
        <widget class="QLabel" name="labelFiberCount_4">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>用量：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="FanoutCount">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_44">
     <item>
      <widget class="QLabel" name="labelDesc_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>描述信息：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="EditDesc">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
